go/cmd/chunk_sink/session-source-handler.go

```diff
@@ -41,20 +42,51 @@ func NewSessionSourceHandler(sessionStorage storage.Storage, recorderUpdateCh ch
 
 // Called after a session has been closed and rendered by storage. Setup above in the constructor
 func (h *SessionSourceHandler) onSessionClosed(session *storage.Session) {
-	// TODO: Refine this
+	// Generate presigned URLs for session files (24 hour expiry)
+	expiry := 24 * time.Hour
```

Comment:
This should be the same as the lifetime of the session. And if the keep flag is set, if thould be forever

Reply:
It's just the download URL